packages/components/src/components/positions-drawer-card/positions-drawer-card.tsx

```diff
@@ -15,40 +15,37 @@ import {
     isMobile,
     isVanillaContract,
 } from '@deriv/shared';
-import { TContractInfo, TContractStore } from '@deriv/shared/src/utils/contract/contract-types';
+import { TContractStore } from '@deriv/shared/src/utils/contract/contract-types';
 import { TToastConfig } from '../types/contract.types';
 
-type TGetEndTime = Pick<
-    TContractInfo,
-    'is_expired' | 'sell_time' | 'status' | 'tick_count' | 'underlying' | 'contract_id'
-> &
-    Required<Pick<TContractInfo, 'date_expiry' | 'contract_type' | 'exit_tick_time' | 'is_path_dependent'>>;
-
 type TPositionsDrawerCardProps = {
     addToast: (toast_config: TToastConfig) => void;
     className?: string;
-    contract_info: TGetEndTime;
+    contract_info: ProposalOpenContract &
+        Portfolio1 & {
+            contract_update?: ContractUpdate | undefined;
+        };
     contract_update: ContractUpdate;
```

Comment:
@maryia-deriv, another NIT, if you make this `contract_update?: ContractUpdate;`, then you could probably change [this](https://github.com/binary-com/deriv-app/pull/7514/commits/cad3ca476ee1f013366b59cf40355f2d8876122b#diff-c09530966087c0752a0801430d97a626ffecd1c8c7f1df88c755d0a032e59aa2R214) from `contract_update={contract_update as ContractUpdate}` to `contract_update={contract_update}`